starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,69 @@
+h1 {
+  font-size: 60px;
+  text-align: center;
+  color: black;
+}
+
+.intro {
+  font-size: 20px;
+  text-align: center;
+  margin-top: 3%;
+  margin-bottom: 2%;
+}
+
+hr {
+  width: 90%;
+  margin-top: 15px;
+  margin-bottom: 15px;
+}
+
+.titles {
+  margin-left: 37px;
+	font-size: 24px;
+	margin-top: 15px;
+	margin-bottom: 15px;
+}
+
+/* PICTURES */
+.recipe {
+  width: 37%;
+  margin: 0 auto;
+  display: flex;
+}
+
+.cooking {
+  width: 32%;
+  margin: 0 auto;
+  display: flex;
+}
+/* PICTURES */
+
+.container {
+  max-width: 75%;
+  margin: 0 auto;
+}
+
+.hero-section {
+  min-height: 500px;
+  background-image: url(../images/apple-pie.jpg);
+  background-size: cover;
+  background-repeat: no-repeat;
+  display: flex;
```

Comment:
Since only half of the image is visible by using your settings, you can try something like this to fit the whole block:
```
background: url("../images/apple-pie.jpg") no-repeat center center;
height: 500px;
background-size: 100%;
```